DQM/L1TMonitorClient/python/L1TStage2uGTEmulatorClient_cff.py

```diff
@@ -21,43 +21,53 @@
     binomialErr = cms.untracked.bool(True)
 )
 
-l1tStage2uGTEmulatorCompRatioClientBXP1 = l1tStage2uGTEmulatorCompRatioClientBX0.clone()
-l1tStage2uGTEmulatorCompRatioClientBXP2 = l1tStage2uGTEmulatorCompRatioClientBX0.clone()
-l1tStage2uGTEmulatorCompRatioClientBXM1 = l1tStage2uGTEmulatorCompRatioClientBX0.clone()
-l1tStage2uGTEmulatorCompRatioClientBXM2 = l1tStage2uGTEmulatorCompRatioClientBX0.clone()
+l1tStage2uGTEmulatorCompRatioClientBXP1 = l1tStage2uGTEmulatorCompRatioClientBX0.clone(
+inputNum  = ugmtEmuDqmDir+'/'+errHistNumStr,
+inputDen  = ugmtEmuDqmDir+'/'+errHistDenStr,
+ratioName = ratioHistStr
+)
+
+l1tStage2uGTEmulatorCompRatioClientBXP2 = l1tStage2uGTEmulatorCompRatioClientBX0.clone(
+inputNum  = ugmtEmuDqmDir+'/'+errHistNumStr,
+inputDen  = ugmtEmuDqmDir+'/'+errHistDenStr,
+ratioName = ratioHistStr
+)
+
+l1tStage2uGTEmulatorCompRatioClientBXM1 = l1tStage2uGTEmulatorCompRatioClientBX0.clone(
+inputNum  = ugmtEmuDqmDir+'/'+errHistNumStr,
+inputDen  = ugmtEmuDqmDir+'/'+errHistDenStr,
+ratioName = ratioHistStr
+)
 
+l1tStage2uGTEmulatorCompRatioClientBXM2 = l1tStage2uGTEmulatorCompRatioClientBX0.clone(
+inputNum  = ugmtEmuDqmDir+'/'+errHistNumStr,
+inputDen  = ugmtEmuDqmDir+'/'+errHistDenStr,
+ratioName = ratioHistStr
+)
 
 BX            = 'BX1'
```

Comment:
WIth the change, these BX assignments are being ignored, please consider them

Reply:
When I consider 
BX = 'BX1'
errHistNumStr = 'dataEmulSummary_' + BX
errHistDenStr = 'normalizationHisto'
ratioHistStr  = 'dataEmulMismatchRatio_' + BX

inside clone of l1tStage2uGTEmulatorCompRatioClientBXP1, there was a compilation error. That's why I didn't consider them. May I know your thoughts on this?

Reply:
Why **inside** the `clone`? It must stay **before** the (relevant) `clone`

Reply:
Yes, thanks! Let me modify it.

Reply:
Yes, otherwise it produces changes like those we are seeing